File: 1848/1848.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var minimumDistanceToTheTargetElem = function(root) {
    if (!root) return 0;
    return 1 + Math.max(minimumDistanceToTheTargetElem(root.left), minimumDistanceToTheTargetElem(root.right));
};

File: 1848/1848.cs
public class Solution {
    public int MinimumDistanceToTheTargetElem(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(MinimumDistanceToTheTargetElem(root.left), MinimumDistanceToTheTargetElem(root.right));
    }
}